src/odemis/driver/actuator.py

```diff
@@ -2806,3 +2806,332 @@ def _doReference(self, future, axes):
                 raise CancelledError()
             future._running_subf = self._lensz.reference(axes)
         future._running_subf.result()
+
+class LinkedAxesActuator(model.Actuator):
+    """
+    The goal of this wrapper is to automatically adjust the underlying stage movement based on the movement of its
+    axes. As the sample is tilted by ~45° (along Y), whenever moving the stage in Y, the distance of the sample to
+    the optical objective changes. This wrapper compensate for this change by linearly mapping the dependant axes.
```

Comment:
Would be maybe nice to add here in what reference frame you will provide the movements to the user of the wrapper.